.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,77 @@
+# Contributing to the CAF provider
+
+👍🎉 First off, thanks for taking the time to contribute! 🎉👍
+
+## Code of conduct
+
+This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
+For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
+contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+
+## What should I know before I get started?
+
+The current goal of the CAFprovider is to support the CAF landing zones but can also be used to standardize the naming convention of your projects. It is important to keep in mind that many of the design decisions on the provider have been made to accommodate the needs of CAF.
+
+To contribute to this project you are required to have at least go 1.13 installed in your system
+
+## Adding a new resource
+
+The process to add a new resource start with checking if the resource has already been implemented.
+
+1. Check if the resource has been implemented already
+
+    You can find a list of resources implemented in the [README.md](../README.md) under the resource status section.
+
+2. Create an issue for the missing resource
+
+    If there is no [issue created already](https://github.com/aztfmod/terraform-provider-azurecaf/issues) for the implementation of this resource you should [create an issue](https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/creating-an-issue) requesting the implementation of the resource.
+
+3. Check the requirements for your resource Name
+
+    You can check the requirements for your resource name in the [docs](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/resource-name-rules) or by trying to create the resource on azure putting and invalid naming and checking the error message
```

Comment:
`by checking the error message returned when trying to create the resource on Azure with an invalid name`